lib/core/src/Cardano/CLI.hs

```diff
@@ -1120,43 +1098,34 @@ cmdVersion = command "version" $ info cmd $ mempty
                             Commands - 'stake-pool'
 -------------------------------------------------------------------------------}
 
-cmdStakePool
-    :: ToJSON apiPool
-    => StakePoolClient apiPool
-    -> Mod CommandFields (IO ())
+cmdStakePool :: StakePoolClient ApiStakePool -> Mod CommandFields (IO ())
 cmdStakePool mkClient =
     command "stake-pool" $ info (helper <*> cmds) $ mempty
         <> progDesc "About stake pools"
   where
-    cmds = subparser $ mempty
-        <> cmdStakePoolList mkClient
+    cmds = subparser $ mempty <> cmdStakePoolList mkClient
 
 -- | Arguments for 'stake-pool list' command
 data StakePoolListArgs = StakePoolListArgs
     { _port :: Port "Wallet"
     , _stake :: Maybe Coin
     }
 
-cmdStakePoolList
-    :: ToJSON apiPool
-    => StakePoolClient apiPool
-    -> Mod CommandFields (IO ())
+cmdStakePoolList :: StakePoolClient ApiStakePool -> Mod CommandFields (IO ())
 cmdStakePoolList mkClient =
-    command "list" $ info (helper <*> cmd) $ mempty
-        <> progDesc "List all known stake pools."
+    command "list" $ info (helper <*> cmd) $
+        mempty <> progDesc "List all known stake pools."
   where
     cmd = fmap exec $ StakePoolListArgs
         <$> portOption <*> stakeOption
-    exec (StakePoolListArgs wPort stake) = do
+    exec (StakePoolListArgs wPort stake) =
         runClient wPort Aeson.encodePretty $ listPools mkClient (ApiT <$> stake)
 
 {-------------------------------------------------------------------------------
                             Commands - 'network'
 -------------------------------------------------------------------------------}
 
-cmdNetwork
-    :: NetworkClient
-    -> Mod CommandFields (IO ())
+cmdNetwork :: NetworkClient -> Mod CommandFields (IO ())
```

Comment:
Hi @Unisay 

I know this is a draft (so hope you can forgive me for commenting randomly! 🙏🏻 ), however:

Is this a formatting change?

While I'm generally in favour of fixing formatting issues, I wonder if it would be easier to review (and rebase, if base changes) if we move formatting changes to a separate PR, so we would have:

1. A PR that just moves files around, without changing anything (unless strictly necessary).
2. A PR that performs formatting fixes.

Reply:
Yeah, makes sense. But you're right that I didn't apply much discipline exactly because this is a dry-run (draft) PR.